docker/Dockerfile.gui

```diff
@@ -1,21 +1,17 @@
-ARG CUDA_VERSION=11.4.0-cudnn8-runtime-ubuntu20.04
-ARG DEEPLABCUT_VERSION=2.2.1.1
-FROM deeplabcut/deeplabcut:${DEEPLABCUT_VERSION}-base-cuda${CUDA_VERSION}
+ARG CUDA_VERSION
+ARG DEEPLABCUT_VERSION
+
+FROM deeplabcut/deeplabcut:${DEEPLABCUT_VERSION}-base-cuda${CUDA_VERSION}-latest
```

Comment:
Let's a deprecation notice to this dockerfile given it is currently not pushed (as a comment)

```suggestion
# NOTE: This dockerfile is currently not included in the build process
# It is still left for reference, but currently untested.
ARG CUDA_VERSION
ARG DEEPLABCUT_VERSION

FROM deeplabcut/deeplabcut:${DEEPLABCUT_VERSION}-base-cuda${CUDA_VERSION}-latest
```